### PR TITLE
modlinks

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

ported from tg at kapus request

## Why It's Good For The Game

they are cool and epic

## Changelog

:cl:
add: MODsuits can now call other modsuits with the MODlink feature
/:cl:

### CODE REVIEW

File: code/__DEFINES/mod.dm [hunk 1]
```diff
@@ -36,5 +36,13 @@
...
 #define MOD_WORN_ICON_OVERRIDE "mod_worn_icon_override"
 
+//Defines for MODlink frequencies
+#define MODLINK_FREQ_NANOTRASEN "NT"
+#define MODLINK_FREQ_SYNDICATE "SYND"
+#define MODLINK_FREQ_CHARLIE "CHRL"
+#define MODLINK_FREQ_CENTCOM "CC"
+
 /// Global list of all /datum/mod_theme
 GLOBAL_LIST_INIT(mod_themes, setup_mod_themes())
```
Comment (Kapu1178) [inline code/__DEFINES/mod.dm:46]: This shouldn't be statically initialized, please do this in the world/New called proc for setting up datum lists.

File: code/game/atoms.dm [hunk 2]
```diff
@@ -2218,3 +2218,25 @@
...
+/// The degree of the opacity is optional, based off the opacity arg (0 -> 1)
+/atom/proc/makeHologram(opacity = 0.5)
+	// First, we'll make things blue (roughly) and sorta transparent
+	add_filter("HOLO: Color and Transparent", 1, color_matrix_filter(rgb(125,180,225, opacity * 255)))
+	// Now we're gonna do a scanline effect
+	// Gonna take this atom and give it a render target, then use it as a source for a filter
+	// (We use an atom because it seems as if setting render_target on an MA is just invalid. I hate this engine)
+	var/atom/movable/scanline = new(null)
+	scanline.icon = 'icons/effects/effects.dmi'
+	scanline.icon_state = "scanline"
```
Comment (Kapu1178) [inline code/game/atoms.dm:2233]:
```suggestion
	var/atom/movable/scanline = new /atom/movable{icon = 'icons/effects/effects.dmi'; icon_state = "scanline"}(null)
```

**Review by Kapu1178 — CHANGES_REQUESTED** [review]
You gotta learn how to use the enter key why do you always write all your code like its an essay 😭

File: code/__DEFINES/mod.dm (hunk 1 above)
Comment (Fikou) [reply code/__DEFINES/mod.dm:46]:
```suggestion
GLOBAL_LIST_EMPTY(mod_themes)
```

Comment (Fikou) [reply code/__DEFINES/mod.dm]: ok i guess

**Comment by Kapu1178** [comment]
yolo

**Pull request merged**